src/components/hand-poses.js

```diff
@@ -11,13 +11,31 @@ const POSES = {
 
 const NETWORK_POSES = ["allOpen", "thumbDown", "indexDown", "mrpDown", "thumbsUp", "point", "allGrip", "pinch"];
 
-AFRAME.registerComponent("hand-pose", {
+AFRAME.registerComponent("hand-pose-state", {
   multiple: true,
   schema: {
     pose: { default: 0 }
   },
+  init() {
+    this.setSelfAsStore = this.setSelfAsStore.bind(this);
+  },
+  play() {
+    this.el.addEventListener("model-loaded", this.setSelfAsStore);
+  },
+  pause() {
+    this.el.removeEventListener("model-loaded", this.setSelfAsStore);
+  },
+  setSelfAsStore(e) {
+    const poseEl = e.target.querySelector(`[hand-pose__${this.id}]`);
+    poseEl.components[`hand-pose__${this.id}`].store = this;
```

Comment:
it seems that there should be a way to avoid this component having to set this back reference -- ideally it shouldn't care about who is asking it for its state. can't the pose behavior be given a selector to use to look this up at startup time and then just stash the reference to it?

Reply:
This was tricky so I wasn't sure what's right here. I couldn't think of how to use querySelector on hand-pose to set the back reference, since I don't know a unique selector each networked-instantiated avatar.
One way is to check for the `networked-avatar` behavior on `this.el`, then on its parentEl, then continue up the parent hierarchy until I find it. Do you think this preferable over making the `networked-avatar` (here it's `hand-pose-state` but that will change in my next commit) set the back reference?

Reply:
I crawled up the parent hierarchy to set the back reference on `hand-pose` because I don't know of a selector to use that will work in the case of network-instantiated players.

Reply:
some quick googling yielded this (I didn't know about it either) -- perhaps the contract can be you pass a selector in and then call `closest()` on the node once it is initialized to find the source of the data?

https://developer.mozilla.org/en-US/docs/Web/API/Element/closest